fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -271,6 +273,9 @@ public Response createOrReplaceObjectRdf(
 
         if (httpConfiguration.putRequiresIfMatch() && StringUtils.isBlank(ifMatch) && !resource.isNew()) {
             throw new ClientErrorException("An If-Match header is required", 428);
+        } else if (resource.hasType(FEDORA_PAIRTREE)) {
+            throw new ForbiddenResourceModificationException("Graphs cannot be created for resources " +
+                                                              "under pairtree nodes");
```

Comment:
Personally, I would prefer wording along the lines of, '"Resources cannot be created under pairtree elements."'